scripts/guichan.lua

```diff
@@ -790,7 +790,7 @@ function WarMenu(title, background, resize)
     end
     bg = ImageWidget(bgg)
   end
-  if (menu:getWidth() == Video.Width) then
+  if (resize == nil or resize == true) then
```

Comment:
uncertain why menu:getWidth() == Video.Width wasn't working (all menus were passing that condition, even the main menu, Load Mods, which is not fullscreen).  It worked when i first did this a week ago against an older codebase.  Anyway, using the same condition here as a few lines above gets it to work.  Fullscreen menu BGs are put into 4:3 and offset accordingly, while non fullscreen menus have their BG drawn at 0,0 (again, see Load Mods menu)